src/ZEO/asyncio/client.py

```diff
@@ -547,7 +547,7 @@ def verify(self, server_tid):
     def get_peername(self):
         return self.protocol.get_peername()
 
-    def call_async_threadsafe(self, future, method, args):
+    def call_async_threadsafe(self, future, _, method, args):
```

Comment:
Wouldn't we be better off naming the argument `wait_ready`?  Or do you have a linter which is complaining about unused arguments?